ci/scripts/run_tests.py

```
@@ -142,55 +140,59 @@ def run_one(
+            cmd.append("--run_slow")
+        if run_integration:
+            cmd.append("--run_integration")
+        if enable_junit:
+            cmd.append(f"--junitxml={junit}")
+        if enable_coverage:
+            # always include nat module in the coverage report
+            cmd.append("--cov=nat")
+            # if the project has a src directory, include it in the coverage report
+            source_dir = project_dir / "src"
+            if source_dir.exists():
+                cmd.append(f"--cov={str(source_dir)}")
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)